CatoNetworks/cato-sase/tests/cato_sase_antimalware_events.json

```diff
@@ -0,0 +1,54 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "cato-sase",
+        "dialect_uuid": "469bd3ae-61c9-4c39-9703-7452882e70da"
+      }
+    },
+    "message": "{\"account_id\":\"1714\",\"action\":\"Block\",\"dest_ip\":\"10.64.4.10\",\"dest_is_site_or_vpn\":\"Site\",\"dest_port\":\"22\",\"event_count\":\"1\",\"event_sub_type\":\"IPS\",\"event_type\":\"Security\",\"internalId\":\"M9w5A3mkAa\",\"ip_protocol\":\"TCP\",\"mitre_attack_subtechniques\":\"\",\"mitre_attack_tactics\":\"\",\"mitre_attack_techniques\":\"\",\"os_type\":\"OS_UNKNOWN\",\"pop_name\":\"Dublin\",\"risk_level\":\"Medium\",\"rule\":\"3605\",\"rule_id\":\"3605\",\"signature_id\":\"feed_ips15_ssh\",\"src_country\":\"China\",\"src_ip\":\"61.177.173.13\",\"src_is_site_or_vpn\":\"Site\",\"src_port\":\"47046\",\"src_site\":\"Reflector\",\"threat_name\":\"IP reputation based signature - Network Scanner\",\"threat_reference\":\"https://support.catonetworks.com/hc/en-us/articles/360011568478\",\"threat_type\":\"Reputation\",\"time\":\"1650596005910\",\"traffic_direction\":\"INBOUND\",\"event_timestamp\":\"2022-04-22T02:53:25Z\"}"
+  },
+  "expected": {
+    "message": "{\"account_id\":\"1714\",\"action\":\"Block\",\"dest_ip\":\"10.64.4.10\",\"dest_is_site_or_vpn\":\"Site\",\"dest_port\":\"22\",\"event_count\":\"1\",\"event_sub_type\":\"IPS\",\"event_type\":\"Security\",\"internalId\":\"M9w5A3mkAa\",\"ip_protocol\":\"TCP\",\"mitre_attack_subtechniques\":\"\",\"mitre_attack_tactics\":\"\",\"mitre_attack_techniques\":\"\",\"os_type\":\"OS_UNKNOWN\",\"pop_name\":\"Dublin\",\"risk_level\":\"Medium\",\"rule\":\"3605\",\"rule_id\":\"3605\",\"signature_id\":\"feed_ips15_ssh\",\"src_country\":\"China\",\"src_ip\":\"61.177.173.13\",\"src_is_site_or_vpn\":\"Site\",\"src_port\":\"47046\",\"src_site\":\"Reflector\",\"threat_name\":\"IP reputation based signature - Network Scanner\",\"threat_reference\":\"https://support.catonetworks.com/hc/en-us/articles/360011568478\",\"threat_type\":\"Reputation\",\"time\":\"1650596005910\",\"traffic_direction\":\"INBOUND\",\"event_timestamp\":\"2022-04-22T02:53:25Z\"}",
+    "event": {
+      "kind": "event",
+      "action": "Block"
```

Comment:
```suggestion
      "action": "block",
      "category": ["malware"],
      "type": ["info"]
```

Reply:
Hm we have 
`event_sub_type` as `IPS` and action as `Block`
Maybe it should be `intrusion_detection` with `denied`?

Reply:
You are right. `intrusion_detection` is more suitable.